src/test/java/io/github/wimdeblauwe/errorhandlingspringbootstarter/handler/SpringSecurityApiExceptionHandlerTest.java

```diff
@@ -64,7 +64,7 @@ void testForbiddenViaGlobalSecurityConfig() throws Exception {
         mockMvc.perform(get("/test/spring-security/admin-global"))
                .andExpect(status().isForbidden())
                .andExpect(header().string("Content-Type", "application/json;charset=UTF-8"))
-               .andExpect(jsonPath("code").value("ACCESS_DENIED"))
+               .andExpect(jsonPath("code").value("AUTHORIZATION_DENIED"))
```

Comment:
I may be missing something here, but I can't see why you would expect these tests to return different codes

1. `testForbiddenViaSecuredAnnotation`
2. `testForbiddenViaGlobalSecurityConfig`

They both try to access an endpoint without the required role, so shouldn't they return the same result in both cases? The only difference is the mechanism by which the endpoints are secured: `@Secured` in the case of (1) and configuration in the case of (2), but I wouldn't expect this to influence the response.